README.md

```diff
@@ -14,12 +20,14 @@ Notes:
 - It is recommended you use either OSX or Linux (Ubunutu is recommended) for development
 - It is recommneded you install the Zephyr base at the root of your user (`~/zephyrproject`)
 
+See the [Beyond Getting Started Guide](https://docs.zephyrproject.org/3.0.0/guides/beyond-GSG.html) for more information on the setup process. 
+
 ## Installing a 3rd Party Toolchain
 
-Reference: https://docs.zephyrproject.org/latest/getting_started/toolchain_3rd_party_x_compilers.html#gnu-arm-embedded
+Reference: https://docs.zephyrproject.org/3.0.0/getting_started/toolchain_3rd_party_x_compilers.html
 
 Going to use the GNU arm embedded toolchain since it supports ARM based SoCs.
-Install gcc-arm-none-eabi-10-2020-q4-major from the GNU website. 
+Install [gcc-arm-none-eabi-10-2020-q4-major](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads/product-release) from the GNU website. 
```

Comment:
No because this can be done for MacOS as well, however, I'm gonna wait for the others try out a new set of steps for MacOS and if they can confirm that those work for them then I'll update this accordingly.